src/core_plugins/elasticsearch/lib/patch_kibana_index.js

```diff
@@ -25,6 +25,12 @@ export async function patchKibanaIndex(options) {
 
   const rootEsType = getRootType(kibanaIndexMappingsDsl);
   const currentMappingsDsl = await getCurrentMappings(callCluster, indexName, rootEsType);
+
+  // patchKibanaIndex() should do nothing if there are no current mappings
```

Comment:
__nit:__ would you mind adding a simple unit test to make sure we handle `404` properly and don't fail unexpectedly?

Reply:
https://github.com/elastic/kibana/pull/14271/commits/ee08eb27861aa9471cd15b34db04cc7dc0c70133